orbit/cmd/orbit/orbit.go

```diff
@@ -350,6 +356,13 @@ func main() {
 			return fmt.Errorf("cleanup old files: %w", err)
 		}
 
+		log.Info().Msg("running single query (SELECT uuid FROM system_info)")
+		uuidStr, err := getUUID(osquerydPath)
+		if err != nil {
+			log.Error().Msg("Error getting uuid " + err.Error())
```

Comment:
I think we should just return the error here `return fmt.Errorf("get UUID: %w", err)`. Because if we don't have the UUID then the whole system won't work as expected.

Reply:
Makes sense, thanks, updated